### ISSUE TITLE
Add "block this site/image" context menu entry

### ISSUE THREAD

**Issue by qsniyg** [qsniyg/maxurl#127]
[As mentioned by @decembre](https://github.com/qsniyg/maxurl/issues/14#issuecomment-532192788), this would be quite useful, some kind of similar idea to uBlock (although simpler).

**Comment by afterdelight** [comment]
There is already exist an extension with similiar function
https://chrome.google.com/webstore/detail/image-blocker-ex%20-9000/eaabjlbddgjhjfplckkiglekpmonabcm

**Comment by qsniyg** [comment]
@afterdelight Sorry, I didn't explain it clearly in the title. It's actually more to add the image/site to the blacklist, which (depending on your options), does one or both of these:

 * Prevents finding larger images from that website ([this was the original use case](https://github.com/qsniyg/maxurl/issues/75), as the person wanted to only download the smaller version, not the larger one, for specific websites)
 * Prevents the popup from loading for that image/website (mostly useful when the popup is set to open automatically when your mouse is over the image, similar to extensions like MPIV and Imagus)

You can already add entries to the blacklist manually, but adding an option to block by right clicking on an image might help make it easier to do so :)

**Comment by qsniyg** [comment]
Would also be useful as a button or key ([reddit comment](https://www.reddit.com/r/MaxImage/comments/ghztsc/auto_max_zoom_that_fits_to_screen_setting/fqgvmbd/))

**Comment by aeltee** [comment]
I came to the github and searched issues for this same feature request. There are somewhat frequent occasions where a popup is triggered when I did not intend it to, for example sometimes when hovering over the background (which seemingly does not even contain any image) of a site. When I look to see the URL of said popped up image, it is usually some semi-related image host. An example of this happening is on: https://chrome.google.com/webstore/detail/view-image/jpcmhcelnjdmblfmjabdeclccemkghjk. When you hover over the white area above the Overview info section, this fox image pops up: https://lh3.googleusercontent.com/snILF0BadWxBSX-keABjIUui55ZbaJLwbYDlhkcrDn5f3Aypikl18Ve33F8lC-aWeoozy4BmXxC-igT0puQJILBpgg=s0?imgmax=0. When you hover over the reviews or related buttons, this unrelated image pops up: https://lh3.googleusercontent.com/85lBtPdOk2iyXKA0pjGdYIxWvoXGXv9qL00pwcwzAK2JGMAenm4xTPLArQPG2Sij4e9YkjjkOf80NfuLRYVUBFiCSw=s0?imgmax=0. 
Long story short, while I could find the URL of the image and then manually add to blacklist, it would be very convenient.
Thanks!
